base/applications/mspaint/imgarea.cpp

```diff
@@ -74,28 +74,30 @@ LRESULT CImgAreaWindow::OnSize(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
     int imgYRes = imageModel.GetHeight();
     sizeboxLeftTop.MoveWindow(
                0,
-               0, 3, 3, TRUE);
+               0, GRIP_SIZE, GRIP_SIZE, TRUE);
     sizeboxCenterTop.MoveWindow(
-               Zoomed(imgXRes) / 2 + 3 * 3 / 4,
-               0, 3, 3, TRUE);
+               GRIP_SIZE + Zoomed(imgXRes - GRIP_SIZE) / 2,
```

Comment:
Why would you zoom the grip size? This doesn't make sense to me.

Reply:
Fixed.